core/src/test/resources/org/apache/calcite/test/RelOptRulesTest.xml

```diff
@@ -11751,7 +11751,7 @@ LogicalProject(DEPTNO=[$0], EMPNO=[$2])
     <Resource name="planAfter">
       <![CDATA[
 LogicalProject(DEPTNO=[$0], EMPNO=[$2])
-  LogicalSort(offset=[2], fetch=[10])
+  LogicalSort(fetch=[10])
```

Comment:
@rubenada , I believe this is the unit test that shows the error. The plan was wrong before the batch, and the fix results in the right plan.

Reply:
I meant a unit test that verified the query result, not the query plan.

Reply:
I'm not very familiar with the code base and unit/int tests available. Are there existing tests I can modify that already check the results of a query?